scripts/certificate-check

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+HOST_ROLE=$(cat /etc/xensource/pool.conf)
```

Comment:
I think it would be safer to do:
```
if grep -q master /etc/xensource/pool.conf; then
..
fi
```
This protects against slight changes in that file, like a space, newline or something else that a `==` would miss

Reply:
I think wwe want to use `-qv` to invert the match